gui/RetroChessChatWidgetHolder.cpp

```diff
@@ -111,29 +111,40 @@ void RetroChessChatWidgetHolder::chessnotify(RsPeerId from_peer_id)
 
 void RetroChessChatWidgetHolder::chessPressed()
 {
-	RsPeerId peer_id =  mChatWidget->getChatId().toPeerId();//TODO support GXSID
-	if (rsRetroChess->hasInviteFrom(peer_id))
-	{
+	ChatId chatId = mChatWidget->getChatId();
+	if (chatId.isDistantChatId()) {
+		RsGxsId distId = chatId.toDistantChatId();
+		rsRetroChess->requestGxsTunnel(distId);
```

Comment:
Incorrect: if the chat id is a distant chat Id, then there is already a tunnel ;-)